src/index.tsx

```diff
@@ -727,6 +730,14 @@ export class Resizable extends React.PureComponent<ResizableProps, State> {
     if (!this.state.isResizing || !this.resizable || !this.window) {
       return;
     }
+    if (event instanceof TouchEvent) {
+      try {
```

Comment:
@Voakie Does following ever fail?

Reply:
Probably not, but it did throw an error before I passed the required options to `addEventListener`. I guess you can remove that but I'm not 100% sure.

Reply:
Nevermind, I remember why you need the try/catch block. It will fail on `mousemove` events.